docs/data-sources/equinix_fabric_cloud_router.md

```diff
@@ -25,15 +25,16 @@ data "equinix_fabric_cloud_router" "cloud_router_data_name" {
 
 ### Optional
 
+- `project` (Block Set) Project information (see [below for nested schema](#nestedblock--project))
```

Comment:
How does a FCR lookup by project work? Is there a one FCR resource per project limitation? Does the datasource error if there are more than one FCR in the selected project? Is that a reasonable launch filter that will need to be augmented with additional filters in the future (name, location, feature based?)

Looking a little closer, it appears that `project` is not used when looking up FCR resources, only the `uuid` is used.

Reply:
I think what is intended is that the datasource would have `uuid` required.

Reply:
Great catch. Agree completely. Will update.